tests/test_awkward.py

```diff
@@ -13,3 +13,13 @@ def test_to_awkward():
     arr = pylhe.to_awkward(pylhe.read_lhe_with_attributes(TEST_FILE))
     assert len(arr) == 791
     assert len(arr.eventinfo.nparticles) == len(arr.particles)
+
+
+def test_awkward_registration():
+    """Awkward registration should correctly happen on import.
+
+    Make sure vector helper classes function properly by calling
+    a property provided by vector `mass` on each particle's momentum.
+    """
```

Comment:
```suggestion
```

My point earlier was that _if_ a test needs comments then they should go in the docstring. This is a simple test and the name alone makes it clear what is happening. You wrote a good test, so this is fine as is. :+1: